src/brogue/Rogue.h

```diff
@@ -2854,7 +2869,7 @@ extern "C" {
     short passableArcCount(short x, short y);
     void analyzeMap(boolean calculateChokeMap);
     boolean buildAMachine(enum machineTypes bp,
-                          short originX, short originY,
+                          pos origin,
                           unsigned long requiredMachineFlags,
                           item *adoptiveItem,
                           item *parentSpawnedItems[50],
```

Comment:
Should we update the array size so it matches the declaration in Architect.c?  I get a compiler warning about mismatched bounds.

Reply:
Done. Clang 14.0.3 doesn't complain about it for some reason, not even with `-Wextra`.